lib/turn.rb

```diff
@@ -0,0 +1,21 @@
+
+class Turn
+  attr_reader :guess, :card
+  def initialize (guess, card)
+    @guess = guess
+    @card = card
+  end
+
+  def correct?
+    if @guess == @card.answer
+      true
+    else
+      false
+    end
+  end
```

Comment:
you could slim up this method by just having `@guess == @card.answer` since this will automatically return true or false for you.